webhooks.md

```diff
@@ -503,6 +503,59 @@ curl -X POST \
   }'
 ```
 
+`transaction.reimbursement.status` event is triggered when a reimbursement changes state on a transaction.
+
+The payload for these events includes the `reimbursement` object with the up-to-date status, and you can read more about it in the [Reimbursement API](/reimbursement/#webhook) documentation.
```

Comment:
I would break this down to two sentences: "The payload ... up-to-date status. Read more about it ..."

Also, I would say "Read more about it on the Reimbursement page." When you say "reimbursement documentation", it sounds like the link will go to some other place, not just to a different page on this website.

Reply:
Thank you, updated the text based on the suggestions